events.json

```diff
@@ -36,5 +36,9 @@
   "SIO_MISSING_PROJECTION_ERROR" : "v1/SIO_MISSING_PROJECTION_ERROR",
   "SIO_SHP_MISMATCHED_GEOMETERY_TYPE_ERROR" : "v1/SIO_SHP_MISMATCHED_GEOMETERY_TYPE_ERROR",
   "SIO_GEOJSON_INVALID_ATTRIBUTE_TYPE_ERROR" : "v1/SIO_GEOJSON_INVALID_ATTRIBUTE_TYPE_ERROR",
-  "SIO_GEOJSON_INVALID_FILE_ERROR" : "v1/SIO_GEOJSON_INVALID_FILE_ERROR"
+  "SIO_GEOJSON_INVALID_FILE_ERROR" : "v1/SIO_GEOJSON_INVALID_FILE_ERROR",
+  "SIO_GEOPKG_INVALID_FILE_ERROR" : "v1/SIO_GEOPKG_INVALID_FILE_ERROR",
+  "SIO_GEOPKG_FEATURE_MISSING_GEOM_ERROR" : "v1/SIO_GEOPKG_FEATURE_MISSIN_GEOM_ERROR",
+  "SIO_GEOPKG_SCHEMA_MISSING_GEOM_ERROR" : "v1/SIO_GEOPKG_SCHEMA_NO_GEOM_ERROR",
+  "SIO_FGDB_INVALID_ZIP_FILE_ERROR" : "v1/SIO_FGDB_INVALID_ZIP_FILE_ERROR",
```

Comment:
You need to remove the comma on the last entry here. This is causing the code generator to fail to generate valid Java, and I believe it's not valid JSON.